README.md

```diff
@@ -1024,6 +1024,8 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 * SQL [[sql](https://crates.io/keywords/sql)]
   * Generic
     * [launchbadge/sqlx](https://github.com/launchbadge/sqlx) - async PostgreSQL/MySQL/SQLite connection pool with strong typing support [![build badge](https://img.shields.io/github/workflow/status/launchbadge/sqlx/Rust/master?style=flat-square)](https://github.com/launchbadge/sqlx)
+  * Generic
```

Comment:
This should be under the other "Generic" block just above here, as opposed to making a new one